torchdata/nodes/_populate_queue.py

```diff
@@ -0,0 +1,40 @@
+# pyre-unsafe
+import queue
+import threading
+from typing import Iterable
+
+from torchdata.nodes.exception_wrapper import ExceptionWrapper, StartupExceptionWrapper
+
+
+def _populate_queue(
+    source: Iterable,
+    q: queue.Queue,
```

Comment:
This should be `Union[queue.Queue, mp.Queue]` ?

Reply:
good catch!